scripts/install-sealedsecrets.sh

```diff
@@ -1,13 +1,11 @@
 #!/bin/bash
 
-GOOS=$(go env GOOS)
-GOARCH=$(go env GOARCH)
-
 release=$(curl -sI https://github.com/bitnami-labs/sealed-secrets/releases/latest | grep Location | awk -F"/" '{ printf "%s", $NF }' | tr -d '\r')
 #release=$(curl --silent "https://api.github.com/repos/bitnami-labs/sealed-secrets/releases/latest" | sed -n 's/.*"tag_name": *"\([^"]*\)".*/\1/p')
 
 echo "SealedSecrets release: $release"
 
-kubectl create -f https://github.com/bitnami-labs/sealed-secrets/releases/download/$release/sealedsecret-crd.yaml
+helm del --purge ofc-bootstrap
```

Comment:
Please remove and move this to `reset.sh`

Reply:
This is more a workaround than part of reset. I couldn't figure out the issue with helm, but it's failing with `Error: a release named ofc-bootstrap (or whatever I put there) already exists.` in a new cluster.